iroha_client/tests/add_asset.rs

```diff
@@ -31,12 +31,10 @@ async fn client_add_asset_quantity_to_existing_asset_should_increase_asset_amoun
     );
     iroha_client
         .submit(create_asset.into())
-        .await
         .expect("Failed to prepare state.");
-    task::sleep(Duration::from_millis(
+    thread::sleep(Duration::from_millis(
```

Comment:
I thought that it would be better to use task sleep inside our tests (except thread sleep after we spawn new thread).

Reply:
We had sync clients now, got it.